capnpc-go/templates/interfaceClient

```diff
@@ -37,6 +59,17 @@ func ({{$.Node.Name}}) DecodeFromPtr(p capnp.Ptr) {{$.Node.Name}} {
 	return {{$.Node.Name}}(capnp.Client{}.DecodeFromPtr(p))
 }
 
+// IsValid reports whether c is a valid reference to a capability.
+// A reference is invalid if it is nil, has resolved to null, or has
+// been released.
 func (c {{$.Node.Name}}) IsValid() bool {
 	return capnp.Client(c).IsValid()
 }
+
+// Update the flowcontrol.FlowLimiter used to manage flow control for
+// this client. This affects all future calls, but not calls already
+// waiting to send. Passing nil sets the value to flowcontrol.NopLimiter,
+// which is also the default.
+func (c {{$.Node.Name}}) SetFlowLimiter(lim flowcontrol.FlowLimiter) {
```

Comment:
Maybe also expose GetFlowLimiter?

Reply:
I was wondering about that.  Is there any reason the user should want to get a flow limiter?  I was thinking this is more of an internal operation that doesn't necessarily need to be exported.

Reply:
I guess it just seems odd to expose one and not the other. But I don't have *that* strong of an opinion, so I'll defer if you feel strongly.

Reply:
Nah, I'm just being lazy.  I'll add it.